kong/plugins/oauth2/access.lua

```diff
@@ -422,7 +422,11 @@ local function parse_access_token(conf)
       parameters[ACCESS_TOKEN] = nil
       ngx.req.set_uri_args(parameters)
 
-      if ngx.req.get_method() ~= "GET" then -- Remove from body
+      local content_type = req_get_headers()[CONTENT_TYPE]
+      local is_upload = content_type and
+        string_find(content_type:lower(), "multipart/form-data", 1, true)
```

Comment:
no need for `lower()` here

> Since the 0.6.9 release, all the header names in the Lua table returned are converted to the pure lower-case form by default, unless the raw argument is set to true (default to false).